components/profile/ProfileActivitySummery.vue

```diff
@@ -123,65 +101,14 @@ useLazyAsyncData('stats', async () => {
     return
   }
 
+  const listEvents: number[] = data.value.listed
```

Comment:
add types to `listed` and `obtained.totalCount`

Reply:
Hey @preschian 

great suggestion

following this suggestion, I think it's a good idea to typify the various graphql queries we have

i have started in this PR

and will create a follow up issue

Reply:
resolving conversation...

Reply:
no worries, I also create similar issue https://github.com/kodadot/nft-gallery/issues/10185